webpack.common.js

```diff
@@ -0,0 +1,59 @@
+const webpack = require("webpack");
+const path = require("path");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const AssetsPlugin = require("assets-webpack-plugin");
+
+module.exports = {
+  entry: {
+    main: path.join(process.cwd(), "src/index.js")
```

Comment:
`process.cwd()` is the default context, so you can just use the relative paths for both `entry` and `output`.

Reply:
I forgot webpack requires abs paths, so I updated it to use the pattern from master `path.join(__dirname, "src", "index.js")`